docs/UserGuide.md

```diff
@@ -120,325 +127,416 @@ Displays the output of `View` command, which expands each ingredient or recipe c
 
-### 4.1.1 Help
+[\[Back to Table of Contents\]](#table-of-contents)
+
+#### 4.2.2 Command Summary
+
+Action | Format
+--------|------------------
+[**Add Ingredient**](#441-add-ingredient) | `add ingredient -n <name> -q <quantity>[<units>] -e <expiry date> [-d <description>]`
+[**Delete Ingredient**](#442-delete-ingredient) | `delete ingredient <index>...`
+[**Edit Ingredient**](#443-edit-ingredient) | `edit ingredient <index> [-n <name>] [-q <quantity> [<units>]] [-e <expiry date>] [-d <description>] [-t <tags>]...`
+[**Clear Ingredient**](#444-clear-ingredient) | `clear ingredient [expired]`
+[**Find Ingredient**](#445-find-ingredient) | `find ingredient <keyword>...`
+[**List Ingredient**](#446-list-ingredient) | `list ingredient`
+[**View Ingredient**](#447-view-ingredient) | `view ingredient <index>`
+[**Add Recipe**](#451-add-recipe) | `add recipe -n <name> -i <ingredient>... [-d <description>] [-s <steps>]...`
+[**Delete Recipe**](#452-delete-recipe) | `delete recipe <index>...`
+[**Edit Recipe**](#453-edit-recipe) | `edit recipe <index> [-n <name>] [-i <ingredient>]... [-d <description>] [-s <steps>]...`
+[**Clear Recipe**](#454-clear-recipe) | `clear recipe`
+[**Find Recipe**](#455-find-recipe) | `find recipe <keyword>...`
+[**List Recipe**](#456-list-recipe) | `list recipe`
+[**View Recipe**](#457-view-recipe) | `view recipe <index>`
+[**Cook Recipe**](#458-cook-recipe) | `cook recipe <index>`
+
+[\[Back to Table of Contents\]](#table-of-contents)
+
+### 4.3 General Commands
+
+#### 4.3.1 Help
 **Format:**<br />
 `help`
 
 Pops out a window that leads the user to [User Guide](https://ay2122s1-cs2103t-w11-1.github.io/tp/UserGuide.html)
 (You are here).
 
-### 4.1.2 Exit
+#### 4.3.2 Exit
 **Format:**<br />
 `exit`
 
 Closes the window and exits the program. All your information will be saved.
 
-## 4.2 Ingredients
+[\[Back to Table of Contents\]](#table-of-contents)
 
-This sections covers commands related to Inventory management. Any command primarily interacting with ingredients
-will be here.
+### 4.4 Ingredients
+This section covers commands related to Inventory management. Any command primarily interacting with ingredients will 
+be here.
+- Note that by default, Fridgy will sort all Ingredients by expiry dates in descending order i.e. soonest expiring
+  item will be at the top.
+- If 2 ingredients' expiry dates are the same, they will be sorted in alphabetical order.
 
-### 4.2.1 Add Ingredients
+#### 4.4.1 Add Ingredient
 
 Add an ingredient to the Inventory.
 
 **Format:**<br />
-`add ingredient -n <name> -q <quantity>[<units>] [-d <description>] -e <expiry date> [-t <tags>]`
+`add ingredient -n <name> -q <quantity> [<units>] -e <expiry date> [-d <description>] [-t <tags>]...`
 
 **Example(s):**<br />
-1. `add ingredient -n tomato -d from africa -q 500 -e 27-09-2021 -t sweet`
-    <br />Expected Output:<br />
-    ![addCommand1.png](images/ingredientCommands/addCommand1.png)
+1. All command flags used:<br />
```

Comment:
Minimal flags first then all command flags like recipe(?)

Reply:
You meant you want us to swap it to match the ordering? Okay. 🐒

Reply:
ye